script/travis/install_boost.sh

```diff
@@ -65,7 +65,7 @@ then
 fi
 # Select the libraries required.
 ALPAKA_BOOST_B2+=" --with-test"
-if [ "${ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE}" == "ON" ]
+if [[ ! -v ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE || "${ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE}" == "ON" ]]
```

Comment:
Same as above, why is fiber installed if the variable `ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE` is not set?

Reply:
Same as above